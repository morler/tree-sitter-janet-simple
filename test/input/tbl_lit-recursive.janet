@{:ant @{:legs 6} :bee @{:legs 6} :cheetah @{:legs 4}}
