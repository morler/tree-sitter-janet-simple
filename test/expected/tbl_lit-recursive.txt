(source [0, 0] - [1, 0]
  (tbl_lit [0, 0] - [0, 54]
    (kwd_lit [0, 2] - [0, 6])
    (tbl_lit [0, 7] - [0, 17]
      (kwd_lit [0, 9] - [0, 14])
      (num_lit [0, 15] - [0, 16]))
    (kwd_lit [0, 18] - [0, 22])
    (tbl_lit [0, 23] - [0, 33]
      (kwd_lit [0, 25] - [0, 30])
      (num_lit [0, 31] - [0, 32]))
    (kwd_lit [0, 34] - [0, 42])
    (tbl_lit [0, 43] - [0, 53]
      (kwd_lit [0, 45] - [0, 50])
      (num_lit [0, 51] - [0, 52]))))
